lookout/style/format/feature_utils.py

```diff
@@ -39,6 +39,19 @@ def __init__(self, value: str, start: Position, end: Position,
         self.y = y
         self.path = path
 
+    @property
+    def is_accumulated_indentation(self) -> bool:
+        """
+        Check if node represents virtual node with accumulated indentation.
+
+        :return: True if the node represents accumulated indentation else False.
+        """
+        indentation = {" ", "\t"}
```

Comment:
probably it should be from imported classes - not to hardcode in many different places